### PR TITLE
Go math ceil

### PR DESCRIPTION
**Pull request by Nessvah**
### Description

Closes issue #2438

I have created a new entry about the Go Ceil math function on a new folder and a new file called ceil.md on the go math functions directory.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import "math"
+
+roundedNumber := math.Ceil(floatingNumber)
+```
+
+The `Cei()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the argument rounded to the nearest integer.
+
+- When passing `Ceil(±Inf)` the returned value will be the same as the argument.
+- When passing `Ceil(Nan)` the returned value will be also `Nan`.
+- When passing `Ceil(±0)` the returned value will be exactly the same as the argument.
```
Comment (CBID2) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:30]: [Typo]
```suggestion
- When passing `Ceil(±Inf)`, the returned value will be the same as the argument.
- When passing `Ceil(Nan)`, the returned value will be also `Nan`.
- When passing `Ceil(±0)`, the returned value will be exactly the same as the argument.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Nessvah! :) I pointed out some typos that could be improved. Other than that, you're off to a good start! :)

**Comment by Nessvah** [comment]
> Hi @Nessvah! :) I pointed out some typos that could be improved. Other than that, you're off to a good start! :)

Thank you! I'll be working on it asap.

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Syntax
+
+```pseudo
+import "math"
+
+roundedNumber := math.Ceil(floatingNumber)
+```
+
+The `Cei()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the argument rounded to the nearest integer.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:26]:
```suggestion
The `Cei()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the next highest integer equal to or greater than the argument.
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    result := math.Ceil(score)
+    fmt.Print(score, " rounded up is ", result, ".")
+    }
+```
+
+## Codebyte Example
+
+The example below is runnable and shows how `Ceil()` handles negative numbers.
+
+```golang
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:56]:
````suggestion
```codebyte/golang
````

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The `Cei()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the argument rounded to the nearest integer.
+
+- When passing `Ceil(±Inf)`, the returned value will be the same as the argument.
+- When passing `Ceil(Nan)`, the returned value will be also `Nan`.
+- When passing `Ceil(±0)`, the returned value will be exactly the same as the argument.
+
+## Example
+
+To use the `Ceil()` function, the `math` package needs to be imported.
+Below is a simple example where the result will be the score, rounded up to the nearest integer.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:35]:
```suggestion
Below is a simple example where the result will be the score, rounded up to the next highest integer.
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+---
+Title: 'Ceil()'
+Description: 'Returns the rounded up value of a given decimal number.'
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:3]:
```suggestion
Description: 'Returns a given decimal number rounded up to the next highest integer.'
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **Ceil()** function returns the rounded up value of a given decimal number.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:16]:
```suggestion
The **Ceil()** function returns a given decimal number rounded up to the next highest integer.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Nessvah I did a first review and left a few comments.

**Comment by Nessvah** [comment]
@SSwiniarski Hi! I applied the suggested changes. Let me know if there is anything else that I can improve on this.

**Review by SSwiniarski — APPROVED** [review]
@Nessvah Looks good for a second maintainer review.

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **Ceil()** function returns a given decimal number rounded up to the next highest integer.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:16]:
```suggestion
The **`Ceil()`** function returns a given decimal number rounded up to the next highest integer.
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+## Syntax
+
+```pseudo
+import "math"
+
+roundedNumber := math.Ceil(floatingNumber)
+```
+
+The `Cei()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the next highest integer equal to or greater than the argument.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:26]:
```suggestion
The `Ceil()` function receives a value of type `float64` and returns a single value of the same type. This returned value is the next highest integer equal to or greater than the argument.
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+- When passing `Ceil(±Inf)`, the returned value will be the same as the argument.
+- When passing `Ceil(Nan)`, the returned value will be also `Nan`.
+- When passing `Ceil(±0)`, the returned value will be exactly the same as the argument.
+
+## Example
+
+To use the `Ceil()` function, the `math` package needs to be imported.
+Below is a simple example where the result will be the score, rounded up to the next highest integer.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:35]:
```suggestion
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+```go
+package main
+
+import (
+    "math"
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:41]:
```suggestion
    "math"  // Required
```

File: content/go/concepts/math-functions/terms/ceil/ceil.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+    "fmt"
+    )
+
+func main(){
+    score := 18.54
+    result := math.Ceil(score)
+    fmt.Print(score, " rounded up is ", result, ".")
+    }
+```
+
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/ceil/ceil.md:51]: There should be a line such as "The output is as follows:" and a shell block with the output

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Nessvah I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by Nessvah** [comment]
@caupolicandiaz hi! Sorry, I took too long to reply. The changes are already in place but since I was busy this past week with a presentation, I forgot to push it!  I'll be doing it later today!

**Review by caupolicandiaz — APPROVED** [review]
@Nessvah Looks good to go.

**Comment by SSwiniarski** [comment]
@Nessvah Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/go/math-functions/ceil

image

**Pull request merged**